### DISCUSSION TITLE
car park louvre

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#11407] [General]
![MicrosoftTeams-image](https://user-images.githubusercontent.com/96638521/216010026-a0fd4dfd-b0f4-4187-acff-e62c4dda2824.png)
![image](https://user-images.githubusercontent.com/96638521/216010240-d197a1ee-8d18-4d8e-9b5c-8bf1921ad579.png)
If I am planning to model a louvre in a car park with a defined height and width but blocked so the free area is 60%, I want to model louvre to a full height and not use a free area assumption. Any technique how can this be applied to restrict flow through an area. I have tried window screen but it is defined for thin obstruction.

**Comment by drjfloyd** [comment]
The discussion forum used by the communtity is here: https://groups.google.com/g/fds-smv. Not many montior the github discussion page.

You can use HVAC for this. Define a VENT with SURF_ID='HVAC' that is the full area of the louver. The duct connected to the louver would be given the free area. You can account for the flow restriction by specifying an appropriate LOSS.

**Reply by ShehabGamrah** [reply]
Thanks. Would that be one vent or two vents connected by nodes and a duct. and also which setting defines flow loss

**Reply by drjfloyd** [reply]
It depends on what your are modeling in your domain. If you aren't looking to model what happens outisde the car park and just need to have the louver vent air from inside,  then you only need one vent and the external duct node could be an AMBIENT node. If you do want to see what happens with the gasses after they leave the garage, then yes you would need a VENT on each side of the wall.

There is a LOSS input on the HVAC input.  If you know the details of the louver you may be able to find a maunfacturers technical speification that tells you the pressure drop which would let you compute the loss. Otherwise you could treat it like orifice flow where you have a discharge coefficient (often called C_d) and the FDS input for LOSS is equivalent to (1/C_d)^2

**Reply by ShehabGamrah** [reply]
thanks. what if I know the height and width of the louvre and the free area. is there a way to model it full height and specify the frea area.

**Reply by Er9y714** [reply]
> thanks. what if I know the height and width of the louvre and the free area. is there a way to model it full height and specify the frea area.

Free area percentage alone is not sufficient to predict the loss. Look at these two louvres with the same free area percentages. One's loss is more than 2 times of the other. Therefore you need to see the datasheet of the louvre or make a resemblance to another product you know the loss of.

![image](https://user-images.githubusercontent.com/26166281/224417866-ba91774a-a942-4dc8-8455-f4a4b7797b8a.png)

![image](https://user-images.githubusercontent.com/26166281/224418021-5cf0dabb-1d4e-43ba-8c44-3277104f79a2.png)

**Comment by drjfloyd** [comment]
Yes. As I said, the VENT is the full area of the louver and the duct gets the free area.

&VENT ID='INLET',XB=-5,5,100,100,-5,5,SURF_ID='HVAC'/
&HVAC ID='INLET',TYPE_ID='NODE',VENT_ID='INLET',DUCT_ID='LOUVER'/
&HVAC ID='OUTLET',TYPE_ID='NODE',AMBIENT=T,XYZ=0,100,0,DUCT_ID='LOUVER'/
&HVAC ID='LOUVER',TYPE_ID='DUCT',NODE_ID='INLET','OUTLET',AREA=0.0001,LOSS=2,2/

The VENT is 10 m x 10 m but the duct area (which is used to get the velocity used for the loss calculation) is 1 cm x 1 cm.

**Reply by ShehabGamrah** [reply]
Sorry to keep asking about this. well I am confused about this, would the 1cm x 1cm equal the free area, then this is very low compared to the 10m x 10 m, also what are the values of 2 for flow loss. I have tried to use the hvac leak option with two basic vents and a leakage are but there is numerical instability.

**Reply by drjfloyd** [reply]
That was a completely made up example to just show that the area of the VENT and the flow are used in the HVAC calculation do not need to be the same. The 10 m x 10 m would be the louver height and width and the 1 cm x 1 cm the free area. Not realistic values, you would make these whatever they need to be for your specific application. 

By deafult, leakage assumes a flow loss representative of small leakage paths. You would want to set a more reasonable value to represent a louver. Either use the full HVAC approach of defining nodes and a duct or you could assign a discharge coefficient to the leak (see the User's Guide for how to do that).

If you haven't already, you should read the technical reference guide on HVAC to see the equations being solved and how loss is used.

As stated in my prior post, LOSS = 1/C_d^2 where C_d is an orifice discharge coefficient. So a loss of 2 is the same as C_d~0.7. Again I had a made up example, you should figure out on your own via reference material what an appropriate loss is for the specific louver in your application
